### PR TITLE
Changes/jekyll for the win

### PR DESCRIPTION
**Pull request by bcronin**
## Summary

Add a Jekyll-generated static site as a starting point. Most of the PR content is from a manual "fork" of https://github.com/barryclark/jekyll-now. 

Gentle emphasis on this as a "starting point" :) No attempt to get the content exactly right, match typography, or do anything other than get an `index.md` in there from which to move forward!

image

### CODE REVIEW

**Comment by bcronin** [comment]
(Removed the extra "the" in "bindings for ~~the~~ many platforms including" visible in the screenshot)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+> March, 2016: If you're on an old version of Jekyll Now and run into a) build warnings or b) syntax highlighting issues caused by [Jekyll 3 and GitHub Pages updates](https://github.com/blog/2100-github-pages-now-faster-and-simpler-with-jekyll-3-0), just :sparkles:[update your _config.yml](https://github.com/barryclark/jekyll-now/pull/445/files):sparkles: and you'll be set!
```
Comment (bensigelman) [inline README.md]: can we kill this file entirely (or strip it down to something minimal about LightStep)?

Comment (bensigelman) [inline README.md]: Hm, now that I think about it: I like the really basic instructions for running the jekyll server (in this case, just `make dev`). Similar to https://github.com/opentracing/opentracing.github.io/blob/master/README.md#testing-localling-with-jekyll ...

File: about.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+layout: page
+title: About
+permalink: /about/
+---
+
+Some information about you!
```
Comment (bensigelman) [inline about.md]: remove?

**Comment by bensigelman** [comment]
LGTM!

**Comment by bcronin** [comment]
Per your comments, added a commit to remove some of unused files and added a simplified README on what to do to edit the site.

Merging away...

**Pull request merged**